modules/overview/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc

```diff
@@ -15,95 +13,99 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
 
 * A place to encapsulate knowledge from field engineers and spread it to all users.
 
-
-
-
 .Prerequisites
 
 * An administrator account on a running instance of OpenShift 4.
 
+== Creating a project 
 
-.Procedure
-
-. Open the OpenShift web console.
+A project allows to organize and manage different resources on the cluster in an isolated unit. Create a project first to host the *{prod}* Operator.
 
-. To create the `{prod}` project, in the left panel, navigate to the *Home* -> *Projects* section.
+.Procedure
 
-. Click the btn:[Create Project] button.
+. Open the OpenShift web console, in the left panel navigate to the *Home* -> *Projects* section.
 
-. In the *Create Project* pop-up window, enter the project details and validate.
+. Click btn:[Create Project].
+. Enter the project details:
 +
-* *Name*: `{prod-short}`.
+* *Name*:  `{prod-short}`.
 * *Display Name*: `{prod}`.
 * *Description*: `{prod}`.
 
-. To install the *{prod}* Operator, in the left panel, navigate to the *Operators* -> *OperatorHub* section.
+== Installing the *{prod}* Operator
 
-. In the *Filter by keyword* field, type `{prod}`.
+*{prod}* Operator provides with all the resources for running {prod-short} such as PostgreSQL, Keycloak, Registries and the {prod-short} server, as well as configures all these services.
```

Comment:
The 1st line under the heading seems indented because it renders like this:
![image](https://user-images.githubusercontent.com/6623103/91880260-833cf280-ec80-11ea-916c-94c2fd270fdd.png)
I don't see that space here though.

Reply:
i removed it in this commit https://github.com/eclipse/che-docs/pull/1514/commits/7ecfc37589704f88017a9a6807e756646c7a76d3